Xamarin.Forms.ControlGallery.WindowsUniversal/MainPage.xaml.cs

```diff
@@ -127,6 +129,30 @@ void AddNativeControls(NestedNativeControlGalleryPage page)
 				}
 			);
 
+			{
+				var stackLayout = new StackLayout()
+				{
+					VerticalOptions = LayoutOptions.Start,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var label = new Label
+				{
+					Text = "Hello World!",
+					VerticalOptions = LayoutOptions.Center,
+					HorizontalOptions = LayoutOptions.Center
+				};
+
+				var bz56843TextBlock = new TextBlock
```

Comment:
How about if I add a comment that we couldn't figure out why this didn't repro but for expediency sake we're moving on and just we're leaving it in for documentation?